calculator.rb

```diff
@@ -0,0 +1,74 @@
+puts "Welcome to the Calculator Program! You can do any of the following: "
+puts "add (+)"
+puts "subtract(-)"
+puts "multiply(*)"
+puts "divide(/)"
+puts "exponify(**)"
+
+# print "What would you like to do: "
+# choice = gets.chomp
+
+operation = ()
+# math_stuff = ["add", "subtract", "multiply", "divide", "exponify","+", "-", "*", "/", "**"]
+print "What would you like to do: "
+choice = gets.chomp
+ if ["add", "subtract", "multiply", "divide", "exponify","+", "-", "*", "/", "**"].include?(choice)
+   puts 
+ else
+   puts "That is not math. Try again."
+     choice = gets.chomp
+end
+   case choice
```

Comment:
A note about indentation: Sort of related to how we look at and understand scope, we aim to keep anything that is in the same scope indented consistently. The _case_ statement you have here should scoot to the left, along with the _if_ statement above!